runtime/parachains/src/paras_inherent.rs

```diff
@@ -142,15 +142,26 @@ fn signed_bitfields_weight<T: Config>(bitfields_len: usize) -> Weight {
 		.saturating_mul(bitfields_len as Weight)
 }
 
+fn backed_candidate_weight<T: frame_system::Config + Config>(
+	candidate: &BackedCandidate<T::Hash>,
+) -> Weight {
+	<<T as Config>::WeightInfo as WeightInfo>::enter_backed_candidates_variable(
+		candidate.validity_votes.len() as u32,
+	)
+	.saturating_add(if candidate.candidate.commitments.new_validation_code.is_some() {
+		<<T as Config>::WeightInfo as WeightInfo>::enter_backed_candidate_code_upgrade()
```

Comment:
We should max the two, not add since `enter_backed_candidate_code_upgrade` takes into account max validity votes

Reply:
@Lldenaurois

Reply:
will be closed by https://github.com/paritytech/polkadot/pull/4287